scripts/yarn-install-modules.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
```

Comment:
Instead of creating an optional yarn-install-modules file, do you all think it would be better to add a conditional to the install-modules file that will use yarn if it exists. Something like:

```
if ! type "yarn" > /dev/null; then
  yarn 2>&1
else
  npm i 2>&1
fi
```
I use bootstrap.sh a lot to update, so it would be helpful if the install-modules script used yarn.